.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,6 +30,15 @@
 
 :beginner: If you have a lot to say, be more detailed in this section.
 
+# Testing
```

Comment:
Changes are usually tested, this is more about whether they have been fully tested.

```suggestion
# Test coverage
```

If so we could move the documentation items from the checklist to a similar section.

Reply:
I thought the idea was to specify the testing you've done specifically, but "test coverage" kinda sounds like automated tests only, doesn't it? Although the suggestions to the comments and whatnot might make it clear enough :)